### PR TITLE
Revert "Beware of Meaty Ores, for the Meatball Man comes"

### PR DESCRIPTION
**Pull request by jcatjmeow**
Reverts yogstation13/Yogstation#9719
Alterative to #9882 

My fears about the meatball man came true. 
9882 is a half-measure in my opinion.

To restate my view on this thing:

Another lolrandom event that isn't actually fun.
If this hits you, you instadie. And it causes massive damage to random parts of the hull.
Strongly against.

Now, obviously reducing the radius helps immensely, no doubt, but I think it's still too much at any size larger than the standard meteor, and at that point it might as well not even exist. And it's still an insta kill/spaces and ruins your workplace. Not fun and interactive.

"It's a rare chance to happen" isn't a good defense for something being unfun, and it's clearly very damaging when it does happen, and not in a fun way, in a "your round is instantly over and your workplace is unusable" way. Events should break up the monotony of a shift when the antags can't, and not just randomly fuck people over with very little recourse. 

And please please don't point to another event and say "what about this other event" this pr is about meatball man. If you want another event tweaked or changed around or removed, tell me somewhere else.

_Plus it's kinda lrp to have a photorealistic image from a youtube video in the game. I do really like that video and the meatball man, but the standards have to be applied evenly even on things I personally like_ 

[here's the reference in question](https://youtu.be/ErQHVUQ6QCk?t=356), while I really enjoy this video a lot it's not something I believe fits in our station

🆑
rscdel: Removes meatball man
/🆑

### CODE REVIEW

**Review by Partheo — APPROVED** [review]

**Comment by Partheo** [comment]
Not getting my thumbs up if you upvote your own PR

**Comment by jcatjmeow** [comment]
> Not getting my thumbs up if you upvote your own PR

well I like my own pr dont I

**Comment by fluffe9911** [comment]
#9882 got merged

File: code/game/gamemodes/meteor/meteors.dm [hunk 1]
```diff
@@ -134,16 +128,14 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
...
-		timerid = QDEL_IN(src, lifetime)
+	SpinAnimation()
+	timerid = QDEL_IN(src, lifetime)
 	chase_target(target)
 
 /obj/effect/meteor/Bump(atom/A)
 	if(A)
 		ram_turf(get_turf(A))
-		playsound(src.loc, meteorsound, 40, TRUE)
+		playsound(src.loc, meteorsound, 40, 1)
```
Comment (jcatjmeow) [inline code/game/gamemodes/meteor/meteors.dm]:
```suggestion
		playsound(src.loc, meteorsound, 40, TRUE)
```

File: code/game/gamemodes/meteor/meteors.dm [hunk 2]
```diff
@@ -184,7 +176,7 @@ GLOBAL_LIST_INIT(meteorsC, list(/obj/effect/meteor/dust)) //for space dust event
 		var/thing_to_spawn = pick(meteordrop)
 		new thing_to_spawn(get_turf(src))
 
-/obj/effect/meteor/proc/chase_target(atom/chasing, delay = move_delay)
+/obj/effect/meteor/proc/chase_target(atom/chasing, delay = 1)
```
Comment (jcatjmeow) [inline code/game/gamemodes/meteor/meteors.dm:179]:
```suggestion
/obj/effect/meteor/proc/chase_target(atom/chasing, delay = move_delay)
```

**Comment by Ratvar** [comment]
Meatball Man is a worse meteors, and meteors is already a shitty event. So not adding meatball man sounds like a good idea.

**Comment by maxion12345** [comment]
....no

**Pull request merged**